tests/data/author/controls/control_with_components.md

```diff
@@ -0,0 +1,101 @@
+---
+
+# ac-1 - \[Access Control\] Policy and Procedures
+
+## Control Statement
+
+- \[a\] Develop, document, and disseminate to all alert personell:
+
+  - \[1\]  A thorough access control policy that:
+
+    - \[a\] Addresses purpose, scope, roles, responsibilities, management commitment, coordination among organizational entities, and compliance; and
+    - \[b\] Is consistent with applicable laws, executive orders, directives, regulations, policies, standards, and guidelines; and
+
+  - \[2\] Procedures to facilitate the implementation of the access control policy and the associated access controls;
+
+- \[b\] Designate an officer to manage the development, documentation, and dissemination of the access control policy and procedures; and
+
+- \[c\] Review and update the current access control:
+
+  - \[1\] Policy weekly and following all meetings; and
+  - \[2\] Procedures monthly and following conferences.
+
+## Control Implementation Guidance
+
+Do it carefully.
+
+## Control Expected Evidence
+
+Detailed evidence logs.
+
+## Control Guidance
+
+Access control policy and procedures address the controls in the AC family that are implemented within systems and organizations. The risk management strategy is an important factor in establishing such policies and procedures. Policies and procedures contribute to security and privacy assurance. Therefore, it is important that security and privacy programs collaborate on the development of access control policy and procedures. Security and privacy program policies and procedures at the organization level are preferable, in general, and may obviate the need for mission- or system-specific policies and procedures. The policy can be included as part of the general security and privacy policy or be represented by multiple policies reflecting the complex nature of organizations. Procedures can be established for security and privacy programs, for mission or business processes, and for systems, if needed. Procedures describe how the policies or controls are implemented and can be directed at the individual or role that is the object of the procedure. Procedures can be documented in system security and privacy plans or in one or more separate documents. Events that may precipitate an update to access control policy and procedures include assessment or audit findings, security incidents or breaches, or changes in laws, executive orders, directives, regulations, policies, standards, and guidelines. Simply restating controls does not constitute an organizational policy or procedure.
+
+______________________________________________________________________
+
+## What is the solution and how is it implemented?
+
+<!-- Please leave this section blank and enter implementation details in the parts below. -->
+
+______________________________________________________________________
+
+## Implementation a.
+
+Text for default component
+
+### Trestle Component
+
+Text for trestle comp
+
+### This System
```

Comment:
Looking at the labels here two things jump out at me;
1) Should 'This system' be forbidden? (or mutually exclusive) to any content without a header.
2) I think we should enforce that the components are unique per-implementation...

Reply:
I was keeping things flexible - but yes it makes sense to disallow 'This System' completely and require its responses be at the top with no component heading.

Reply:
I was keeping things flexible - but yes it makes sense to disallow 'This System' and require its entries be at the top with no component heading specified.